packages/playground-preview-worker/src/sentry.ts

```diff
@@ -1,5 +1,5 @@
 import { Toucan } from "toucan-js";
-import { z, ZodError } from "zod";
+import { ZodError } from "zod";
```

Comment:
This was a redundant import? Do we not have linting rules to catch this on this package?